native/swift/Tests/wordpress-api/UserAgentTests.swift

```diff
@@ -0,0 +1,33 @@
+import Foundation
+import Testing
+import WordPressAPI
+
+@Suite("User Agent Tests")
+struct UserAgentTests {
+    #if canImport(Darwin)
+    @Test("User agent contains bundle name and version")
+    func testThatDefaultUserAgentContainsBundleNameAndVersion() throws {
+        #expect(UserAgent.postfix.contains(/xctest\/(\d+.\d+)/))
```

Comment:
It's feasible that Apple ships `xctest/x.y.z` at some point, but I figure we can address that down the road if it happens